style.css

```diff
@@ -0,0 +1,161 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
```

Comment:
Please don't target elements using only element names. Increase your specificity by maybe wrapping all your work in a div and giving it a unique id then target like so:
```
#id-name element-name {
prop: some-value
}
```
Apply this method in all your styling in the future to avoid conflicts with other people's styles